PowerEditor/gcc/makefile

```diff
@@ -45,7 +45,7 @@ CPP_DEFINE := UNICODE _UNICODE OEMRESOURCE NOMINMAX _WIN32_WINNT=_WIN32_WINNT_VI
 LD := $(CXX)
 LDFLAGS := -municode -mwindows
 LD_PATH :=
-LD_LINK := comctl32 crypt32 dbghelp ole32 sensapi shlwapi uuid uxtheme version wininet wintrust
+LD_LINK := comctl32 crypt32 dbghelp ole32 sensapi shlwapi uuid uxtheme version wininet wintrust dwmapi
```

Comment:
I would keep it since this is how MS recommends it
https://learn.microsoft.com/en-us/windows/apps/desktop/modernize/apply-windows-themes#enable-a-dark-mode-title-bar-for-win32-applications

Reply:
@ozone10 
Then why don't we use only this API DwmSetWindowAttribute for windows 11/10?
Why do we need to tell 19041U build for using it?

Reply:
@donho 
It's for builds 19041 and greater.

Reply:
@ozone10 
> It's for builds 19041 and greater.

Can't find this info in above link.
Do you have any ref regarding it?

Reply:
@donho 
for Windows 10 it's undocumented, but from Windows 10 20H1 build 19041 it's working same as in Windows 11.

Reply:
@ozone10 
Are you telling me that **Windows 10 20H1 build 19041 (and later builds)** is **Windows 11** except the version number? 
Or you're saying that they have different look & version number, but sharing the same runtime library & Win32 API?

Reply:
@donho 
they use same Win32 API, it is just that this particular function did not have in Windows 10 SDK macro/enum defined. 

There are some win32 API that were undocumented in previous OS and then documented in next OS.

Reply:
@ozone10 
(part 1/2)
According your logic:
> but from Windows 10 20H1 build 19041 it's working same as in Windows 11.

I would remove:
```cpp
bool IsWindows10(DWORD build)
{
	return (IsWindows10() && g_buildNumber >= build);
}
```
and modify `bool IsWindows11(DWORD build)` to:
```cpp
bool IsWindows11(DWORD build)
{
	return (IsWindows10() && g_buildNumber >= build);
}
```
...